### DISCUSSION TITLE
Creating a new user fails - directus.app/files 400 with message "Invalid foreign key."

### DISCUSSION THREAD

**Discussion by Velpre** [directus/directus#16482] [Archive]
Hi 
I have a strange problem that I can't find a solution for. Appreciate some help. 

Problem occurs when uploading a image(file) together with the creation of a new user.

Description:
* If I first log in with one of the user profiles, then log out, and after that create a new user...everything works fine.Image is uploaded and user is created.
* If I DO NOT log in and out with one of the users and immediately after starting the program try to create a new user, I get this error message:

<img width="620" alt="Skjermbilde 2022-11-17 kl  02 09 38" src="https://user-images.githubusercontent.com/65472724/202330960-56b626ba-5e8b-4907-ab00-a7852caed6ab.png">

It is strange that everything works fine if I log in and out but not if i try to make new user-profile right after i start application without logging in and out. 

Anybody who can help me to figure out how to solve this problem?

Thank you

**Comment by rijkvanzanten** [comment]
Can you share the SDK script you're using to do this?

**Reply by Velpre** [reply]
Yes, SDK version is:     10.1.4
